timemachine/ff/handlers/nonbonded.py

```diff
@@ -534,8 +516,12 @@ def static_parameterize(params, smirks, mol):
             SMIRKS patterns matching bonds, to be parsed using OpenEye Toolkits
         mol: Chem.ROMol
             molecule to be parameterized.
-
+        validate: bool
+            check params, smirks
         """
+        if validate:
```

Comment:
Is there a reason this is optional (i.e. isn't the ff invalid if the check fails)?

Reply:
nvm, I see the use case below